lib/libkvm/kvm_open.3

```diff
@@ -245,6 +245,14 @@ The value passed via
 was
 .Dv NULL .
 .El
+.Sh NOTE
+Full memory dumps taken on 13.x (excluding 13.0) and 14.x amd64 kernels
+will cause both
+.Fn kvm_open
+and
+.Fn kvm_open2
+to fail since they do not provide sufficient information to figure out
+where in physical memory the kernel was loaded.
```

Comment:
I might further comment that minidumps are the default (`sysctl debug.minidump`), so this is unlikely to bite people.

We really ought to have a kerneldump.4 man page which documents these kinds of things (what's a "full dump", what's a minidump, what do they look like at a high level, etc. etc.)...

Reply:
> We really ought to have a kerneldump.4 man page which documents these kinds of things (what's a "full dump", what's a minidump, what do they look like at a high level, etc. etc.)...

I think dumpon(8) partly explains this, but yea they should be explained more thoroughly in their own man page.

Reply:
Ah, right, I forgot about dumpon(8). You might consider adding an `.Xr dumpon 8` too, then.